timestamp.go

```diff
@@ -19,6 +19,8 @@ import (
 // to/from BSON
 type Timestamp time.Time
 
+const invalidYear = 1970
```

Comment:
why 1970 is invalid year?

Reply:
While I don't think this would ever happen in Augury, 1970 is a valid date.
Since timestamp starts on the 1970-01-01 I guess we need to use 1969 instead.
I have proposed bellow an implementation that doesn't need the const at all.

Reply:
I agree with @danikp here..
I think that its should be UNIX_EPOCH_TIMESTAMP
https://en.wikipedia.org/wiki/Epoch_(computing)